packages/neuron-ui/src/widgets/Alert/index.tsx

```diff
@@ -8,11 +8,16 @@ export enum AlertStatus {
   Error = 'error',
 }
 
-const Alert: React.FC<{ status: AlertStatus; className?: string }> = ({ status, children, className }) => {
+const Alert: React.FC<{ status: AlertStatus; className?: string; withIcon?: boolean }> = ({
+  status,
+  children,
+  className,
+  withIcon = false,
```

Comment:
Its default value should be true because it does not have this attribution equal `withIcon=true`